x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec

```diff
@@ -3024,3 +3027,17 @@ wkt:keyword                |pt:cartesian_point
 "POINT(111)"               |null
 // end::to_cartesianpoint-str-parse-error-result[]
 ;
+
+stExtentCartesianShapes non-grouping with null shapes
+required_capability: st_extent_agg
+
+# this test ensures that the non-grouping ST_EXTENT_AGG is able to gracefully handle missing cartesian shapes
+
+FROM cartesian_multipolygons
+| STATS extent = ST_EXTENT_AGG(shape), count = COUNT()
```

Comment:
There are other tests with `ST_EXTENT_AGG(shape)` that'll absorb your new `null` lines without needing a new test. Right?

I suppose you could add a `WHERE shape IS NOT NULL` filter here and name it something else. I imagine this is pretty similar for all of your other tests. Right?